src/Stryker.Core/Stryker.Core/Initialisation/ProjectFileReader.cs

```diff
@@ -49,22 +50,30 @@ private string FindProjectReference(XDocument document, string projectUnderTestN
                 else
                 {
                     var searchResult = projectReferences.Where(x => x.ToLower().Contains(projectUnderTestNameFilter.ToLower())).ToList();
-                    if(!searchResult.Any())
+                    if (!searchResult.Any())
                     {
                         throw new ArgumentException($"No project reference matched your --project-file={projectUnderTestNameFilter} argument to specify the project to mutate, was the name spelled correctly?", innerException: new Exception($"Found the following references: {referencesString}"));
                     } else if (searchResult.Count() > 1)
                     {
                         throw new ArgumentException($"More than one project reference matched your --project-file={projectUnderTestNameFilter} argument to specify the project to mutate, please specify the name more detailed", innerException: new Exception($"Found the following references: {referencesString}"));
                     }
-                    return searchResult.Single();
+                    return ConvertPathSeparators(searchResult.Single());
                 }
             }
             else if (!projectReferences.Any())
             {
                 throw new NotSupportedException("No project references found in test project file, unable to find project to mutate.");
             }
-            return projectReferences.Single();
+            return ConvertPathSeparators(projectReferences.Single());
+        }
 
+        private static string ConvertPathSeparators(string filePath)
+        {
+            if (Path.DirectorySeparatorChar == '\\')
+            {
+                return filePath;
+            }
+            return filePath.Replace('\\', Path.DirectorySeparatorChar);
```

Comment:
Could you either add an `else`-statement here or use an elvis-operator to make it more clear that the last return is the else-statement?

Also, it may be nicer to store `'\\'` in a variable at the start of the method to ensure that we use the same value throughout the method.